src/simtools/simtel/simulator_light_emission.py

```diff
@@ -520,7 +892,18 @@ def run_simulation(self, args_dict, figures):
                 stdout=log_file,
                 stderr=log_file,
             )
-        self.process_simulation_output(args_dict, figures)
+        # Append our Python logs to the same logfile for convenience
+        file_handler = logging.FileHandler(Path(self.output_directory) / "logfile.log", mode="a")
+        formatter = logging.Formatter("%(asctime)s - %(levelname)s - %(name)s - %(message)s")
+        file_handler.setFormatter(formatter)
+        file_handler.setLevel(logging.INFO)
+        # Ensure this module logger emits INFO and has a file handler
+        self._logger.setLevel(logging.INFO)
```

Comment:
Need to explain this - why do we change the logging level here. e.g. when the user want to run with the debug option?